### PR TITLE
RSDK-9476: Remove residual debug attribute properties on cameras.

### PR DESCRIPTION
**Pull request by dgottlieb**

### CODE REVIEW

File: components/camera/ffmpeg/ffmpeg.go [hunk 1]
```diff
@@ -26,7 +26,6 @@ import (
 type Config struct {
 	CameraParameters     *transform.PinholeCameraIntrinsics `json:"intrinsic_parameters,omitempty"`
 	DistortionParameters *transform.BrownConrady            `json:"distortion_parameters,omitempty"`
-	Debug                bool                               `json:"debug,omitempty"`
```
Comment (dgottlieb) [inline components/camera/ffmpeg/ffmpeg.go:29]: @randhid, I'm fairly certain this is safe? I wouldn't expect there to be an error if an existing config has `debug: true`. Do you know for sure if this is true? Happy to run an experiment, but I suspect you already have the answer.

Comment (randhid) [reply components/camera/ffmpeg/ffmpeg.go:29]: I have done this before in cowboy attribute deletion mode, yes a complete config removal is not breaking in my experience. Breaking config changes are the addition of a required field and the change of the name of an attribute that is required.

Right now people will have useless debug fields in their configs that do nothing. I'm okay with that.

**Review by randhid — APPROVED** [review]
There is also one in [transform camera](https://github.com/viamrobotics/rdk/blob/baf6c50963ec05ba848e4891deb2b938e1ba25a6/components/camera/transformpipeline/pipeline.go#L65).

**Comment by randhid** [comment]
Ugh, sorry - also in the attributes for [`image_file` camera](https://github.com/viamrobotics/rdk/blob/ebe95fe6f30c658fbea9531871ceb1cb77170c33/components/camera/fake/image_file.go#L69)

**Pull request merged**